example/src/views/AutoCompleteView.vue

```diff
@@ -1,520 +1,449 @@
+          v-bind="objectOmit(autoCompleteProp, ['value'])"
+          :data-cy="`auto-complete-custom-inner-${i}`"
+          :item-height="autoCompleteProp.dense ? undefined : 80"
+          :label-class="autoCompleteProp.dense ? undefined : 'h-20'"
+          variant="outlined"
+        >
+          <template #selection="{ item }">
+            {{ item.id }} | {{ item.label }}
+          </template>
+          <template #item="{ item }">
+            <span
+              class="block"
+              :class="{ 'my-4': !autoCompleteProp.dense }"
+            >
+              {{ item.label }}
+            </span>
+          </template>
+        </RuiAutoComplete>
+      </div>
+    </div>
+    <h4
+      class="text-h6 mt-6"
+      data-cy="auto-completes-custom-options"
+    >
+      Auto Complete: custom options
+    </h4>
+    <div class="grid gap-6 grid-cols-2">
+      <div
+        v-for="(item, i) in autoCompleteCustom"
+        :key="i"
+        class="py-4"
+      >
+        <RuiAutoComplete
+          v-model="item.modelValue"
+          clearable
+          auto-select-first
+          v-bind="objectOmit(item, ['modelValue'])"
+          :append-width="1.5"
+          :data-cy="`auto-complete-custom-options-${i}`"
+        >
+          <template #item.append="{ active }">
+            <RuiIcon
+              v-if="active"
+              class="transition"
+              name="check-line"
+              size="24"
+            />
+          </template>
+        </RuiAutoComplete>
+      </div>
+    </div>
+    <h4
+      class="text-h6 mt-6"
+      data-cy="auto-completes-custom-options"
+    >
+      Auto Complete: primitive values
+    </h4>
+    <div class="grid gap-6 grid-cols-2">
+      <div
+        v-for="(item, i) in autoCompletePrimitive"
+        :key="i"
+        class="py-4"
+      >
+        <RuiAutoComplete
+          v-model="item.modelValue"
+          clearable
+          auto-select-first
+          return-primitive
+          v-bind="objectOmit(item, ['modelValue'])"
+          :append-width="1.5"
+          :data-cy="`auto-complete-custom-options-${i}`"
+        >
+          <template #item.append="{ active }">
+            <RuiIcon
+              v-if="active"
+              class="transition"
+              name="check-line"
+              size="24"
+            />
+          </template>
+        </RuiAutoComplete>
+      </div>
+    </div><h4
+      class="text-h6 mt-6"
+      data-cy="auto-completes-readonly"
+    >
+      Auto Complete: readonly
+    </h4>
+    <div class="grid gap-6 grid-cols-2">
+      <div
+        v-for="(item, i) in autoCompletePrimitive"
         :key="i"
-        v-model="field.modelValue"
-        :data-cy="`auto-complete-${i}`"
-        v-bind="objectOmit(field, ['modelValue'])"
-      />
+        class="py-4"
+      >
+        <RuiAutoComplete
+          v-model="item.modelValue"
+          clearable
+          auto-select-first
+          v-bind="objectOmit(item, ['modelValue'])"
```

Comment:
the `autoCompletePrimitive` (and also the other) definition above still uses `value` instead of `modelValue`, need to be adjusted